torch/testing/_legacy.py

```diff
@@ -16,6 +16,7 @@
     "all_types_and_complex_and",
     "all_types_and_half",
     "complex_types",
+    "complex_types_and",
```

Comment:
We need to remove this entry. Otherwise it will be exposed in the `torch.testing` namespace and will emit a deprecation warning if called:

https://github.com/pytorch/pytorch/blob/28dfed962aad022cde2794ecff08bd6446e42d7d/torch/testing/_deprecated.py#L93-L101

Given that this dtype getter is new, we should just keep it private in the first place.

Reply:
Got it. Thanks @pmeier!